plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -1421,9 +1421,7 @@ public void testTimestampCoercion()
                 .addRoundTrip("TIMESTAMP '1969-12-31 23:59:59.999999499999'", "TIMESTAMP '1969-12-31 23:59:59.999999'")
```

Comment:
Can you please provide specific tests all the options to create table with SHOW CREATE TABLE to verify the DDL?